Tests/XCUITests/SaveLoginsTests.swift

```diff
@@ -215,10 +216,10 @@ class SaveLoginTest: BaseTestCase {
         navigator.goto(LoginsSettings)
         // This only appears the first time
         sleep(2)
+        unlockLoginsView()
```

Comment:
I'm afraid that this is not the solution.. this will only work once the app has run at least once. Not sure why it passed on Bitrise but if we do this, and the biometric screen is shown, the test will fail.. this call has to be after tapping on the Continue button for sure.
What we need to make sure of is that if the biometrics screen appears, we need to wait on it upon clicking on Continue.